el7/Dockerfile

```diff
@@ -34,45 +34,28 @@ RUN yum install -y \
     krb5-server \
     expect
 
-RUN yum groupinstall -y "Development Tools"
+RUN yum groupinstall -y "Development Tools" \
+    && yum install -y centos-release-scl \
+    && yum install -y devtoolset-8-gcc devtoolset-8-gcc-c++
 
-RUN yum install -y centos-release-scl
-RUN yum install -y devtoolset-8-gcc devtoolset-8-gcc-c++
-
-ADD install-openssl11.sh /install-openssl11.sh
-RUN /install-openssl11.sh
+ADD *.sh /
```

Comment:
I'm a bit wary of copying _every script there is_, we might inadvertently copy something we don't need in the build context in the future (and invalidate build layer cache as a result). Also wondering if it's worth to clean all those scripts up in the end?